spec/models/food_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Food do
```

Comment:
So we would want this to be a PORO test vs a Model test. Very similar to how you have it set up in here